ansible/roles/jenkins-worker/templates/ibmi_start.j2

```diff
@@ -0,0 +1,27 @@
+#!/QOpenSys/pkgs/bin/bash
+CIJOBNAME="JENKINS_CI"
+SBMJOB_OPTS="JOB($CIJOBNAME) USER(IOJS)"
+
+export PATH=/QOpenSys/pkgs/lib/ccache:/QOpenSys/pkgs/bin:/QOpenSys/usr/bin:/usr/ccs/bin:/QOpenSys/usr/bin/X11:/usr/sbin:.:/usr/bin
+export PASE_PATH=$PATH
+export JENKS_HOME={{ home }}/{{ server_user }}
+export NODE_COMMON_PIPE="$HOME/test.pipe"
+export NODE_TEST_DIR="$HOME/tmp"
+export JOBS=4
+export OSTYPE=ibmi
+export QIBM_USE_DESCRIPTOR_STDIO=Y
+export HOME=/home/IOJS
```

Comment:
I think this should either
1.  be moved before the `$HOME` references (for `NODE_COMMON_PIPE` and `NODE_TEST_DIR`) above, or
2. we reference `$JENKS_HOME` for `NODE_COMMON_PIPE` and `NODE_TEST_DIR` instead, or
3. we switch to use templates (e.g. `{{ home }}/{{ server_user }}` rather than `$HOME`)